### PR TITLE
add additional exemplars: halo && uptime kuma

### PR DESCRIPTION
**Pull request by bxy4543**
add additional exemplars: halo && uptime kuma

### CODE REVIEW

File: docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# 如何快速安装 Halo 博客平台：
+
+在 Sealos 上快速部署 halo 博客平台 :
```
Comment (yangchuansheng) [inline docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md:3]: Halo

File: docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+### 查看数据库配置：
+
+![img.png](./images/img-5.png)
+
+保存这里生成的配置中包含数据库的用户名密码以及数据库连接方式，用于下文 Halo 中配置：
+
+![img.png](./images/img-6.png)
+
+> 图中可以看到 pgsql 配置如下：
```
Comment (yangchuansheng) [inline docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md:33]: PostgreSQL

File: docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+>
+> Username:root
+>
+> Password: jxidRwmY82eeuFa01tHN28msb86woounM0QMbyl1jhwzKxT9IDqlNkFfyy4R34G3 
+>
+> pg dns: acid-halo-pg.ns-sy32q9p9.svc.cluster.local:5432
```
Comment (yangchuansheng) [inline docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md:39]: DNS name

File: docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+- 镜像名称设置为 `halohub/halo:2.5`
+
+- CPU 和存储应根据实际情况进行配置，memory 需要 `500Mi` 的内存才能启动 Halo，这里可以参考官方 prepare：https://docs.halo.run/getting-started/prepare
+
+- 暴露端口应为 `8090`。同时，设置为外网访问将自动为应用配置一个出口域名用于外网访问，同时需要在 Halo 中进行配置该地址
+
+- 环境变量包括数据库配置和其他相关配置。需要注意的是，你需要根据你在 [步骤 1](#1) 中设置的用户名、密码和 DNS 来配置数据库连接信息。同时，你还需要为 Halo 配置超级管理员的用户名和密码。
+
+配置 Helo 环境变量 :
```
Comment (yangchuansheng) [inline docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md:61]: Halo

File: docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+### **First Halo：**
+
+![img.png](./images/img-16.png)
+
+![img.png](./images/img-17.png)
+
+![img.png](./images/img-18.png)
+
+## FAQ：
```
Comment (yangchuansheng) [inline docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md:130]: FAQ

File: docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+### 忘记密码：
```
Comment (yangchuansheng) [inline docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md:132]: Remove the colon at the end

File: docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+
+#### 进入 launchpad，点击日志查看：
```
Comment (yangchuansheng) [inline docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md:134]: App Launchpad

File: docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
+# 如何快速安装 Halo 博客平台：
```
Comment (yangchuansheng) [inline docs/4.0/i18n/zh-Hans/additional-exemplars/halo/halo.md:1]: 快速安装 Halo 博客平台

File: docs/4.0/i18n/zh-Hans/additional-exemplars/uptime-kuma/uptime-kuma.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
+# 如何快速安装 Uptime Kuma：
```
Comment (yangchuansheng) [inline docs/4.0/i18n/zh-Hans/additional-exemplars/uptime-kuma/uptime-kuma.md:1]: 快速安装 Uptime Kuma

File: docs/4.0/i18n/zh-Hans/additional-exemplars/uptime-kuma/uptime-kuma.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+- 容器暴露端口指定为 3001。
+- 设置为外网访问即可通过给出的域名访问。
+
+![image-20230530103515475](./images/img-3.png)
+
+![image-20230530103515475](./images/img-4.png)
+
+### 步骤 4: 设置存储卷
+
+在高级配置中，添加存储卷挂载来持久化 uptime kuma 数据目录 `/app/data`。这样可以确保应用数据在容器重启后不会丢失。
```
Comment (nowinkeyy) [inline docs/4.0/i18n/zh-Hans/additional-exemplars/uptime-kuma/uptime-kuma.md:28]: Uptime Kuma

**Review by zzjin — APPROVED** [review]

**Pull request merged**